alacritty/src/input.rs

```diff
@@ -1239,7 +1208,8 @@ mod tests {
             },
             window_id: unsafe { std::mem::transmute_copy(&0) },
         },
-        end_state: ClickState::None,
+        end_state: ClickState::Click,
```

Comment:
This change in the test covers the exact issue I've mentioned in the clickstate code. Since there's no clear advantage to this specific behavior I think changing the tests here should be fine though.